homeassistant/components/tautulli/strings.json

```diff
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Tautulli",
```

Comment:
The title is not translatable, as in, this is a name of an application that doesn't change along with other languages.

```suggestion
```